staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -430,7 +430,6 @@ const (
 	OpenAPIV3Patch          FeatureGate = "KUBECTL_OPENAPIV3_PATCH"
 	RemoteCommandWebsockets FeatureGate = "KUBECTL_REMOTE_COMMAND_WEBSOCKETS"
 	PortForwardWebsockets   FeatureGate = "KUBECTL_PORT_FORWARD_WEBSOCKETS"
-	DebugCustomProfile      FeatureGate = "KUBECTL_DEBUG_CUSTOM_PROFILE"
```

Comment:
Based on discussion in https://github.com/kubernetes/enhancements/pull/4824, and to keep it consistent with regular features - don't remove it, add a comment that it should be dropped in 1.34.

Reply:
Thank you for review. Updated.